cli/package.json

```diff
@@ -8,7 +8,7 @@
   },
   "bugs": "https://github.com/Joystream/substrate-runtime-joystream/issues",
   "dependencies": {
-    "@joystream/types": "^0.10.0",
+    "@joystream/types": "./types",
```

Comment:
Yes it would indeed. I think we need to investigate what is the best practice inside a workspace. I did make you change a similar import once with '*' for the same reason :) 
We really need to look into `lerna` and if it perhaps helps to resolve the correct version when at the point when we actually publish. 

@Gamaranto is exploring using it now while working on atlas.

It is worth mentioning also that he used it to build packages in the correct order based on their dependencies. Maybe it could also help to detect circular dependencies?